From: Darik Horn <dajhorn@vanadac.com>
Date: Mon, 27 Feb 2012 20:45:37 -0600
Subject: Add POST_BUILD to the dkms_conf_variables list.

The POST_BUILD directive is ignored by the read_conf() function
because it not in dkms_conf_variables list.
---
 dkms |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- dkms
+++ dkms
@@ -23,7 +23,7 @@
 # All of the variables we will accept from dkms.conf.
 # Does not include directives
 readonly dkms_conf_variables="CLEAN REMAKE_INITRD remake_initrd PACKAGE_NAME
-       PACKAGE_VERSION POST_ADD POST_INSTALL POST_REMOVE PRE_BUILD
+       PACKAGE_VERSION POST_ADD POST_BUILD POST_INSTALL POST_REMOVE PRE_BUILD
        PRE_INSTALL BUILD_EXCLUSIVE_KERNEL BUILD_EXCLUSIVE_ARCH
        build_exclude OBSOLETE_BY MAKE MAKE_MATCH MODULES_CONF
        modules_conf_array PATCH PATCH_MATCH patch_array BUILT_MODULE_NAME
-- 
